src/Components/ProposalStatus/ProposalStatus.test.tsx

```diff
@@ -1,12 +1,12 @@
 import ProposalStatus from './ProposalStatus';
 import { render } from '../../utils/tests';
 import { ProposalStatusProps } from './types';
-import { ContractState } from 'types/types.guilds.d';
+import { ProposalState } from 'types/types.guilds.d';
 import moment from 'moment';
 
 const validProps: ProposalStatusProps = {
   timeDetail: 'Time',
-  status: ContractState.Active,
+  status: ProposalState.Active,

```

Comment:
Not sure why we are reverting this

Reply:
I think the change is correct, since the `status` prop that gets passed to the `<ProposalStatus>` component are defined in `ProposalState` type.

In this particular example makes no difference since both ContractState.Active and ProposalStatus.Active are the same, but potentially the component can have an `Executable` state, and that state isn't present in the `ContractState` type.

Reply:
Ts compiler was complaining about that since ProposalStatusProps interface was defined with `ProposalState` enum, not ContractState for status.